src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/Privacy/form.html.twig

```diff
@@ -9,7 +9,13 @@
 
         {% for message in app.flashes('error') %}
             <div class="message error">
-                {{ message }}
+                {{ message|trans }}
+            </div>
+        {% endfor %}
+
+        {% for message in app.flashes('notice') %}
+            <div class="flashMessage info">
```

Comment:
<div class="message error">
vs 
<div class="flashMessage info">

Does the styling look OK?
I would expect it to be: 
<div class="message info">, because the message contains the styling in the error flash?

Reply:
I think Johan's suggestion is very good. And it makes the class declarations in sync with the Symfony standard.

However ;) in our previous sprints Koen created components that are user friendly from an a11y standpoint. Those flash messages are styled slightly different. See the flash message stylesheet for some pointers on how to render them. Also consider implementing the close button. https://github.com/SURFnet/sp-dashboard/blob/066a1b5a6b77927233643a1d219cee5ff088257b/app/scss/components/flash_message.scss